scripts/actions/send-and-update-translation-queue.js

```diff
@@ -0,0 +1,200 @@
+const fs = require('fs');
+const path = require('path');
+const FormData = require('form-data');
+const fetch = require('node-fetch');
+
+const loadFromDB = require('./utils/load-from-db');
+const saveToDB = require('./utils/save-to-db');
+const serializeMDX = require('./serialize-mdx');
+const { vendorRequest, getAccessToken } = require('./utils/vendor-request');
+
+/**
+ * @typedef {Object} Page
```

Comment:
It was something I was playing around with. I'm happy to rip this stuff out if it's obnoxious.

I _did_ find it helpful to pick up this code after longer absences because I could look at the typing for hints as to what was going on.

I'd be super down for using typescript, but I know that's not a decision to be made lightly.

Reply:
I was half joking haha. That being said, if we find types super helpful, its definitely worth exploring.